cypress/integration/legacy_to_beta_spec.js

```diff
@@ -37,17 +36,38 @@ const createDatasetOnLegacy = () => {
   cy.get('#next-button').click()
 
   cy.contains('h3', 'Publisher')
-  cy.get('#owner_org').select('Ministry of Justice')
+  cy.get('#owner_org').select('Land Registry')
   cy.get('#save-button').click()
 }
 
 const triggerSync = () => {
-  cy.request('...')
+  cy.request(publishSyncBetaUrl)
+    .then(res => {
+      expect(res.status).to.eq(200)
+    })
+}
+
+const findDatasetOnBeta = (testDatasetName) => {
+  cy.visit(findUrl)
+  cy.get('.dgu-filters__apply-button').click()
+  cy.contains('h1', 'Find government data')
+  cy.get('#q').type(testDatasetName)
+  cy.get('.dgu-search-box__button').click()
+  cy.contains('h2',testDatasetName)
 }
 
 describe('Synchronisation tests', () => {
-  it('synchronises on beta a dataset created on Legacy', () => {
-    createDatasetOnLegacy()
-//    triggerSync()
+  const testDatasetName = `legacy dataset ${new Date().toISOString()}`
```

Comment:
There is an issue with the passing the same testDatasetName to both tests -they do not match, and is the only reason why they are failing at the mo. There is probably a library to deal with this, alternatively we change the testDatasetName to something else.